qa/automation/CareerPortal.Tests.Scenarios/Savostin/buttons.feature

```diff
@@ -0,0 +1,52 @@
+Feature: 1.0.0 buttons lead to correct pages
+Background: 
+	Given "Career Quantori" page is opened
+
+Scenario Outline: 1.0.0_01 All buttons of header lead to correct pages
+	When user clicks on <button>
+	Then <page> is opened
+
+	Examples:
+	|button            |page                                 |
+	|About             |https://career.quantori.com/about    |
+	|Open job positions|https://career.quantori.com/positions|
+	|Contact           |https://career.quantori.com/contact  |
+
+Scenario Outline: 1.0.0_02 All buttons of footer lead to correct pages
+	When user clicks on <button>
+	# VPN must be active
```

Comment:
Why do we need VPN? I can access https://career.quantori.com without VPN

Reply:
We need VPN to open linkedin (the rightmost and lowest button of the page).  This comment about VPN has been updated.